src/neynarAPI/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { mnemonicToAccount } from "viem/accounts";
+
+export async function generateSignature(
+  publicKey: string,
+  appFid: number,
+  accountMnemonic: string,
+  deadline: number
+): Promise<string> {
+  // DO NOT CHANGE ANY VALUES IN THIS CONSTANT
```

Comment:
I assume the order of the parameters matters too?

Reply:
I don't think so, why?
```
const SIGNED_KEY_REQUEST_VALIDATOR_EIP_712_DOMAIN: {
    name: string;
    version: string;
    chainId: number;
    verifyingContract: `0x${string}`;
}
```

Reply:
Oh sorry, my mistake